ui/app/(ai)/analyst/(tools)/overview.ts

```diff
@@ -0,0 +1,48 @@
+import { tool } from "@langchain/core/tools";
+
+import {
+  getFindingsBySeverity,
+  getFindingsByStatus,
+  getProvidersOverview,
+} from "@/actions/overview/overview";
+import {
+  getFindingsBySeveritySchema,
+  getFindingsByStatusSchema,
+  getProvidersOverviewSchema,
+} from "@/types/lighthouse";
+
+export const getProvidersOverviewTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved all tools code to `/lib/lighthouse/tools/` directory